styles.css

```diff
@@ -0,0 +1,86 @@
+:root {
+    --cir-primary-200: #F3EED9;
+    --cir-primary-400: #824936;
+
+     --cir-neutral-100: #FFF;
+    --cir-neutral-900: #222C2A;
+
+    --ff-primary: 'Roboto',sans serif;
+    --ff-accent:'playfair Display',serif;
+}
+body {
+    font-family: var(--ff-primary);
+    font-weight: 400;
+    font-size: 1.3125rem;
+    line-height: 1.6;
+}
+h1,
+h2,
+h3 {
+    line-height: 1;
+    font-family: var(--ff-accent);
+    font-weight: 900;
+    margin: 1rem;
+}
+h2,
+h3,
+p {
+    margin: 1rem;
+}
+img {
+    display: block;
+    max-width:100%;
+}
+.text-center {
+   text-align: center;
+}
+header,
+section {
+    padding: 3rem  0;
+}
+
+/* @media (min-width:40rem) {
+    header,
+section {
+    padding: 4rem  0;
+}
+} */
```

Comment:
please do not leave unused code in the codebase. Reason unused code make your code work not to be clean